x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots.js

```diff
@@ -57,28 +57,36 @@ export function screenshotsObservableFactory(server) {
     // the dashboard is using the `itemsCountAttribute` attribute to let us
     // know how many items to expect since gridster incrementally adds panels
     // we have to use this hint to wait for all of them
-    await browser.waitForSelector(`${layout.selectors.renderComplete},[${layout.selectors.itemsCountAttribute}]`);
+    await browser.waitForSelector(
+      `${layout.selectors.renderComplete},[${layout.selectors.itemsCountAttribute}]`
+    );
   };
 
   const checkForToastMessage = async (browser, layout) => {
     await browser.waitForSelector(layout.selectors.toastHeader, { silent: true });
     const toastHeaderText = await browser.evaluate({
-      fn: function (selector) {
```

Comment:
Great catch -- this is the line that triggered me to think we need a linter for this specific case (no-unbound-methods)

Reply:
oh, LOL. My thinking was to simplify this line, because prettier changed this to `function(selector)` and that failed lint (no space before parens).

Reply:
Ok, so I lied: this PR is not purely just prettier changes. You caught me! Actually, I forgot I made this change 🏸